### ISSUE TITLE
Is it possible to export the config to the project settings?

### ISSUE THREAD

**Issue by LizZerrG** [fruitstudios/craft-searchit#26]
We're using this plugin on our local, testing, staging and production environments. It would be very convenient and time-saving if the config was exported to the project settings, instead of only saved in the database.

**Comment by nickdunn** [comment]
Yes please! Would much prefer to see this synced to project config that be stored in a database table to make it easier to deploy changes across environments.

**Comment by clebuchegger** [comment]
Yes, love the plugin but that's really missing, please implement this, thanks!
